bitnami/mongodb-sharded/templates/config-server/config-server-statefulset.yaml

```diff
@@ -271,6 +271,170 @@ spec:
           {{- end }}
           resources: {{ toYaml .Values.metrics.resources | nindent 12 }}
         {{- end }}
+        {{- range $car := $.Values.configsvr.sidecars }}
```

Comment:
Why not using `tplValue` for the whole array? That would bring much more flexibility than having to set all the cases in the yaml. Users could use the templating language without issues (because it is using tplValue) and the yaml would become more readable